groups/nts/ntsa/ntsa_event.h

```diff
@@ -356,31 +696,37 @@ const ntsa::Error& Event::error() const
 NTSCFG_INLINE
 bool Event::isReadable() const
 {
-    return ((d_state & (1 << e_READABLE)) != 0);
+    return ((d_state & (1U << e_READABLE)) != 0);
 }
 
 NTSCFG_INLINE
 bool Event::isWritable() const
 {
-    return ((d_state & (1 << e_WRITABLE)) != 0);
+    return ((d_state & (1U << e_WRITABLE)) != 0);
+}
+
+NTSCFG_INLINE
+bool Event::isExceptional() const
+{
+    return ((d_state & (1U << e_EXCEPTIONAL)) != 0);
```

Comment:
It is said that 
```
        /// The socket has an error or an exceptional condition.
        e_ERROR = 4,
```

Doesn't it mean that here it is needed also to check `e_ERROR` bit?

Reply:
An e_ERROR does not imply e_EXCEPTIONAL, this is mistake made when revising documentation from NTC. I have fixed this.